triggers/tests/BUILD.bazel

```diff
@@ -104,6 +108,44 @@ client_server_test(
     server_files = ["$(rootpath :acs.dar)"],
 )
 
+da_scala_test_suite(
+    name = "trigger-integration-tests",
+    srcs = glob(
+        ["src/**/*.scala"],
```

Comment:
This duplicates sources between `test_client` and this target. But then this target also depends on `test_client`. Can this duplication be removed, or is this one of the things that should be cleaned up in future PRs?

Reply:
I was planning to clean that up in the end when we can just kill `test_client`. In the meantime it doesn’t seem worth worrying too much about factoring out the code.